packages/react-components/src/organisms/__tests__/UserProfileRole.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+import { UserProfileContext } from '@asap-hub/react-context';
+
+import UserProfileRole from '../UserProfileRole';
+
+const defaultProps = {
+  firstName: 'Phillip',
+  labs: [],
+  teams: [],
+  researchInterests: 'These are my research interest',
+  responsibilities: 'these are my responsibilities',
+};
+
+it('generates a heading', () => {
+  const { getByText } = render(<UserProfileRole {...defaultProps} />);
+  expect(getByText(/role.on.asap/i).tagName).toBe('H2');
+});
+
+it('renders a link to team page', () => {
+  const { getAllByRole } = render(
+    <UserProfileRole
+      {...defaultProps}
+      teams={[
+        {
+          id: '42',
+          displayName: 'Team',
+          role: 'Lead PI (Core Leadership)',
+        },
+      ]}
+    />,
+  );
+
+  const links = (getAllByRole('link') as HTMLAnchorElement[]).map(
+    ({ href }) => href,
+  );
+  expect(links).toMatchInlineSnapshot(`
+    Array [
+      "http://localhost/network/teams/42",
+    ]
+  `);
+});
+
+it('renders responsibilities if present', () => {
+  const { rerender, queryAllByText } = render(
+    <UserProfileRole {...defaultProps} responsibilities={undefined} />,
+  );
+  expect(queryAllByText(/responsibilities/i)).toHaveLength(0);
+
+  rerender(<UserProfileRole {...defaultProps} />);
```

Comment:
You should specifically set something in this rather than relying on the default props